backend/package.json

```diff
@@ -10,29 +10,31 @@
     "start": "node dist/Server.js",
     "test:unit": "jest unit --detectOpenHandles --colors",
     "test:int": "jest int --detectOpenHandles --colors",
-    "test:dev:unit": "jest --watch unit --detectOpenHandles --colors"
+    "test:unit:watch": "jest --watch unit --detectOpenHandles --colors"
```

Comment:
these looks weird tbh. you have repeating `test:dev:unit` and `test:unit`. 
Just to make our life easier can we just write `test` & `test:watch` for `jest unit` where your integration test script will be the same.

Reply:
Do you mean like `"test:unit:watch": "npm run test:unit --watch"` ? Sure, didn't even notice this lol.

Reply:
This one resolved then.